app/views/shared/_data_types.html.erb

```diff
@@ -0,0 +1,7 @@
+<ul class="evidence-types">
+  <% presenter.data_type_keys.each do |data_type| %>
+    <li class="evidence-types__item data-types__<%= "#{data_type.downcase}" %>">
+      <%= render 'shared/evidence_types', icon: presenter.data_type_icon(data_type), text: data_type %>
+    </li>
+  <% end %>
+</ul>
```

Comment:
The shared directory is already full of partials. This partial should be under the evidence summary dir. I think you can leave how it is on Margo PR and then I can move to the right place. Is this okay?